js/main.js

```diff
@@ -34,5 +45,123 @@ function handleSave(event) {
+
   $newEntry.reset();
+
+  $noEntriesMessage.classList.add('hidden');
+  showPage('entries');
+}
+
+function handleNew(event) {
+  showPage('entry-form');
+}
+
+function handleDOMLoad(event) {
+  var previousDataJSON = localStorage.getItem('data-JSON');
+  if (previousDataJSON) {
+    data = JSON.parse(previousDataJSON);
+    showPage(data.view);
+    if (data.entries.length !== 0) {
+      $noEntriesMessage.classList.add('hidden');
+    }
+  }
+
+  for (var i = 0; i < data.entries.length; i++) {
+    $entriesDisplayed.insertBefore(journalEntry(data.entries[i]), $entriesDisplayed.firstElementChild);
+  }
+}
+
+function handleNav(event) {
+  if (!(event.target.matches('a'))) {
+    return;
+  }
+
+  if (event.target.textContent === 'Entries') {
+    event.preventDefault();
+    showPage('entries');
+  }
+}
+
+function handleClose(event) {
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('data-JSON', dataJSON);
+}
+
+function showPage(page) {
+  data.view = page;
+
+  for (var i = 0; i < $views.length; i++) {
+    if ($views[i].getAttribute('data-view') === page) {
+      $views[i].classList.remove('hidden');
+    } else {
+      $views[i].classList.add('hidden');
+    }
+  }
+}
+
+function journalEntry(entry) {
+  // <li class="entries-list-item">
+  //   <div class="row">
+  //     <div class="column-half">
+  //       <img>
+  //     </div>
+
+  //     <div class="column-half">
+  //         <h2 class="entry-heading">title</h2>
+  //         <p class="entry-par">notes</p>
+  //     </div>
+  //   </div>
+  // </li>
+
+  var $li = document.createElement('li');
+  $li.className = 'entries-list-item';
+
+  var $row = document.createElement('div');
+  $row.className = 'row';
+  $li.appendChild($row);
+
+  var $imgCol = document.createElement('div');
+  $imgCol.className = 'column-half';
+  $row.appendChild($imgCol);
+
+  var $image = document.createElement('img');
+  $image.setAttribute('src', entry.url);
+  $image.className = 'max-width rounded img-preview';
+  $imgCol.appendChild($image);
+
+  var $textCol = document.createElement('div');
+  $textCol.className = 'column-half';
+  $row.appendChild($textCol);
+
+  var $entryTitle = document.createElement('h2');
+  $entryTitle.className = 'entry-heading';
+  $entryTitle.textContent = entry.title;
+  $textCol.appendChild($entryTitle);
+
+  var $entryNotes = document.createElement('p');
+  $entryNotes.className = 'entry-par';
+  $entryNotes.textContent = entry.notes;
+  $textCol.appendChild($entryNotes);
+
+  return $li;
+}
+
+function resetPage() {
+  localStorage.clear();
+
+  data = {
+    view: 'entry-form',
+    entries: [],
+    editing: null,
+    nextEntryId: 1
+  };
+
+  for (var i = 3; i < $entriesDisplayed.childNodes.length; i) {
+    console.log('removed node:', $entriesDisplayed.childNodes[1]);
+    $entriesDisplayed.childNodes[1].remove();
+  }
+
+  $noEntriesMessage.classList.remove('hidden');
+
 }
```

Comment:
Is this for testing purposes? If so, you should remove it from the code here since it isn't part of the functionality for the app, you can run this in the browser if you ever need to reset your data